src/core/stdcpp/exception.d

```diff
@@ -85,15 +85,15 @@ else version (CppRuntime_DigitalMars)
     {
     @nogc:
         ///
-        this() nothrow {}
+        extern(D) this() nothrow {}
```

Comment:
Why do we need this at all ?

Reply:
extern(D) or the constructor?

Reply:
The ctor

Reply:
To be able to new these classes ourselves?  There are other modules in core.stdcpp that derive from `exception` that call this constructor.

Reply:
Leaving the line alone for the time being.